src/components/calculator.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
+export default class Calculator extends React.Component {
```

Comment:
- [ ] OPTIONAL:  `export default class Calculator extends Component` is preferred. Also make sure to change the import code to  `import React, { Component } from 'react'` if you want to take this OPTIONAL change into consideration.